app/helpers/rails_db/tables_helper.rb

```diff
@@ -43,5 +43,17 @@ def table_pagination_path
       params.merge({action: :data})
     end
 
+    def column_is_checked?(column_name)
+      if cookies["Table: #{params[:table_id]}"].present?
+        !cookies["Table: #{params[:table_id]}"].include?(" #{column_name}")
+      else
+        true
+      end
+    end
+
+    def display_style(column_name)

```

Comment:
display_style for what? use normal method name, it MUST be clear what this method is doing and for what